### PR TITLE
Update docs for ksonnet

### PR DESCRIPTION
**Pull request by jbeda**
Signed-off-by: Joe Beda <joe.github@bedafamily.com>

### CODE REVIEW

File: docs/TUTORIAL.md [hunk 1]
```diff
@@ -226,37 +226,37 @@ generated JSON itself:
...
+The `ksonnet-lib` project (temporarily) does not currently have a
+single `libsonnet` file. Instead, the project is currently split
 into two files:
 
-* `kubeCore.libsonnet`, which uses Jsonnet's powerful object model and
+* `core.libsonnet`, which uses Jsonnet's powerful object model and
   function support to (concisely) implement the Kubernetes API (both
   [v1][v1] and [v1beta1][v1beta1]); and
-* `kubeUtil.libsonnet`, which is a small collection of helper methods
+* `kubutileUtil.libsonnet`, which is a small collection of helper methods
```
Comment (anguslees) [inline docs/TUTORIAL.md]: This should be `util.libsonnet`

File: README.md [hunk 2]
```diff
@@ -30,7 +28,7 @@ Then, fork or clone this repository, and add the appropriate import
 statements for the library to your Jsonnet code. For example 
 (from the tutorial):
 
-```c++
+```jsonnet
```
Comment (anguslees) [inline README.md]: (Alternatively, consider "javascript")

**Comment by jbeda** [comment]
Accidentally dupped some of @Bradamant3's changes.

**Comment by anguslees** [comment]
@jbeda That's ok, all my typos are someone else's fault too :)

**Comment by jbeda** [comment]
Closing this to not step on @Bradamant3's toes.

**Pull request closed without merging**